packages/core/src/progress-circle/progress-circle.stories.mdx

```diff
@@ -1,7 +1,7 @@
 import { Meta, Props, Story, Preview, API } from '@storybook/addon-docs/blocks';
 import { html, LitElement } from 'lit-element';
 
-<Meta title="Components (Preview)/Circular Progress" />
```

Comment:
If we are moving circular progress out of Preview, should we remove the beta image from the nav on the website?